pkgs/tools/security/wapiti/default.nix

```diff
@@ -0,0 +1,107 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "wapiti";
+  version = "3.0.4";
+
+  src = fetchFromGitHub {
+    owner = "wapiti-scanner";
+    repo = pname;
+    rev = version;
+    sha256 = "0wnz4nq1q5y74ksb1kcss9vdih0kbrmnkfbyc2ngd9id1ixfamxb";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
+    pytest-runner
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    requests
+    beautifulsoup4
+    tld
+    yaswfp
+    Mako
+    pysocks
+    markupsafe
+    six
+    importlib-metadata
```

Comment:
This shouldn't be required in 3.9 anymore.

Reply:
Thanks, that slipped in.